doc/how_to_start.rst

```diff
@@ -0,0 +1,72 @@
+
+.. code:: yaml
+
+    tutorial: true
+    ...
+    retrieve_databundle: true
+
+After doing so, you just need run the modeling workflow with the following command:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake -j 1 solve_all_networks
+
+.. TODO Explain settings of the tutorial case
+
+This command will trigger loading of the whole dataset needed to build the model for a tutorial case that is simulation of power systems in Nigeria and Benin. Note please that data loading and model building will take a while (about 20..50 minutes).
+
+Working run
+------------------------------
+
+After playing with the tutorial model, it's important to clean-up data in your model folder to avoid data conflicts. You may use the `clean` rule for making so:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake -j 1 clean
+
+Generally, it's a good idea to repeat the cleaning procedure every time when the underlying data are changed.
+
+It's recommended to set `retrieve_databundle: true` when building the model first time to download the common data files needed. The load will start automatically when running the model with:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake -j 1 solve_all_networks
+
+Please, note that around **20 Gb zipped files will be downloaded**. It's worth to make sure you have a stable connection, time and around 50 Gb available in your system. If no errors show up, then you can proceed. It's advisable to set `retrieve_databundle: false` after the first model run when all the needed data will be successfully extracted to avoid data loss.
+
+Snakemake
+===========================
+
+Snakemake is a workflow management tool inherited by PyPSA-Earth from PyPSA-Eur. Snakemake decomposes a large software process into a set of subtasks, or ’rules’, that are automatically chained to obtain the desired output.
+
+.. note::
+  ``Snakemake``, which is one of the major dependencies, will be automatically installed in the environment pypsa-earth, thereby there is no need to install it manually.
+
+The snakemake included in the conda environment pypsa-earth can be used to execute any custom rule with the following command:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake < your custom rule >  
+
+Starting with essential usability features, the implemented PyPSA-Earth `Snakemake procedure <https://github.com/pypsa-meets-earth/pypsa-earth/blob/main/Snakefile>`_ that allows to flexibly execute the entire workflow with various options without writing a single line of python code. For instance, you can model the world energy system or any subset of countries only using the required data. Wildcards, which are special generic keys that can assume multiple values depending on the configuration options, help to execute large workflows with parameter sweeps and various options.
+
+
```

Comment:
@AnasAlgarei  is I guess working on a notebook that analysis the tutorial. We definitely should link to the  tutorial notebook of Anas, store somewhere the final network (e.g. zenodo) such that people can play around without solving the previous steps

Reply:
Added the fragment you have suggested. I think it's good first iteration to introduce data analysis. 

Although, I feel it should be improved further:

1) it doesn't sound very supportive to send people to the PyPSA example section... Of course, it's exciting material, fundamentals and excellent read. But if you are interested in regional modeling you are probably looking for information how to tackle you particular regional problem. Absolutely agree that tutorial and learning materials which @AnasAlgarei  is preparing, will be here much more helpful

2) according to our Snakemake, there are some rules intended to results analysis. Can we probably mention it there? (along with a reference to the tutorial materials)

Reply:
On 1., The notebook provided by Anas will be the main material. We only refer to the PyPSA examples such that people can see more examples on how to analyse or modify the network. It can also give ideas of what is not yet implemented in PyPSA but what can potentially be integrated.
1. Notebooks are here https://github.com/PyPSA/PyPSA/tree/master/examples/notebooks
2. These are also linked to the repository e.g. https://pypsa.readthedocs.io/en/latest/examples/lopf_with_pyomo_False.html

On 2., explaining that the Snakefile with a bunch of rule exists is helpful. When can then execute 1-2 rules from there. I think which rule doesn't matter to much. Feel free to choose the most suitable one